pkgs/dart_services/README.md

```diff
@@ -43,6 +43,17 @@ To rebuild the shelf router, run:
 dart run build_runner build --delete-conflicting-outputs
 ```
 
+### Building storage artifacts
+Dart services pre-compiles `.dill` files for the Dart SDK and Flutter Web SDK, which
+are uploaded to Cloud Storage automatically. These files are located in the 
+`artifacts/` directory.
+
+If you need to re-generate these files, run the following command.
+
+```
+grind build-storage-artifacts
```

Comment:
Or, if you don't have `grind` on your path,

```suggestion
dart tool/grind.dart build-storage-artifacts
```

Reply:
okay, I added some additional instructions everywhere